packages/cyberstorm/src/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { Fragment } from "react";
+import styles from "./componentStyles/Breadcrumb.module.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHouse, faSlashForward } from "@fortawesome/pro-regular-svg-icons";
+import { LinkingProvider } from "@thunderstore/components";
+import { LinkLibrary } from "./LinkLibrary";
+
+interface Crumb {
+  label: string;
+  LinkComponent?: (props: any) => React.ReactElement | null; // eslint-disable-line @typescript-eslint/no-explicit-any
+  LinkProps?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+interface Props {
+  parts: Crumb[];
+}
+
+/**
+ * Navigational bread crumbs defining where the user is at the site.
+ */
+export const BreadCrumb: React.FC<Props> = (props) => {
+  if (!props.parts) {
+    return (
+      <div className={`${styles.breadcrumb}`}>
+        <Home />
+      </div>
+    );
+  }
+  const parts = [...props.parts];
+  const leaf = parts.pop();
+
+  return (
+    <div className={`${styles.breadcrumb}`}>
+      <Home />
+      {(parts.length || leaf !== undefined) && <Separator />}
+
+      {parts.map((crumb, i) => (
+        <Fragment key={`${crumb.label}-${i}`}>
+          <MidCrumb crumb={crumb} />
+          <Separator />
+        </Fragment>
+      ))}
+
+      <LeafCrumb crumb={leaf} />
+    </div>
+  );
+};
+
+const Home: React.FC = () => {
+  return (
+    <LinkingProvider value={LinkLibrary}>
+      <LinkLibrary.Index>
+        <FontAwesomeIcon
+          fixedWidth={true}
```

Comment:
It was agreed on Monday's meeting that hard coded boolean props should use short form of `<FontAwesomeIcon fixedWidth ...>`. This is repeated multiple times in the file.

Check out the meeting notes if you half-assed the meeting.